index.html

```diff
@@ -7,33 +7,42 @@
     <meta name="author" content="Daria Pavlyuk" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Daria Pavlyuk Portfolio</title>
+    <link rel  = "preconnect" href ="https://fonts.googleapis.com">
+    <link href ="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+    <link rel ="stylesheet" href = "css/index.css">
+    <link rel ="stylesheet" href ="css/media.css">
   </head>
   <body>
-    <!-- <img src="IMG_5301.jpeg" alt="Personal Photo"> -->
-    <h1 style="color: rgb(130, 207, 238)">Daria Pavlyuk</h1>
+    <ul>
+      <li><a href="#About">About</a></li>
+      <li><a href="#Experience">Experience</a></li>
+      <li><a href="#Connect">Connect</a></li>
+    </ul>
+    <div class = "wrapper">
+    <h1>Daria Pavlyuk</h1>
```

Comment:
To add image, you can the image with the  line of code below
` <img src="./assets/{image-name}.jpg"  alt="computer with lines of code on screen" width="300" height="300" />`

Create an assets folder and add the image there.